content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md

```diff
@@ -0,0 +1,157 @@
+---
+weight: 40
+title: Release 10.16.0.0
+layout: bundle
+---
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>DTM application is integrated with Gainsight to capture the data.</td>
+<td>CTM-130</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Graceful handling of custom property and asset type deletion from backend is implemented.</td>
+<td>CTM-141</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>On delete operation in Property library page confirmation pop-up is shown.</td>
+<td>CTM-159</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Implemeted maximum file size field for custom property of file type.</td>
+<td>CTM-175</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Graceful handling of asset type edit case when it has instances created out of it.</td>
```

Comment:
see above, what does "graceful" mean?

Reply:
Editing an already used AssetType will show a popup on click of save button stating below
"This asset type is used by one or more existing assets and they may not reflect this change. Do you want to proceed?"
User can click on either "Cancel" or "Confirm" button

Reply:
```suggestion
<td>Added a confirmation popup which notifies user that the changes made to asset type will not be reflected in the assets that are currently associated with it</td>
```